WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -2903,6 +2910,11 @@ public void onAddGifClicked(boolean allowMultipleSelection) {
         onPhotoPickerIconClicked(PhotoPickerIcon.GIF, allowMultipleSelection);
     }
 
+    @Override
+    public void onAddFileClicked(boolean allowMultipleSelection) {
+        WPMediaUtils.launchFileLibrary(this, allowMultipleSelection);
```

Comment:
This is a nitpick but could we use the `MediaUtilsWrapper` here and above instead of a static call? This will help to improve testability if we ever need to do this in the future.

Reply:
Hey @jd-alexander, I agree with your suggestion but we have a lot of places where we are already using `WPMediaUtils`. Maybe this change can be separate PR where we can replace all usages of `WPMediaUtils`, what do you think?

Reply:
Yes, that sounds like a good idea. Let's leave this as is then.